test/Microsoft.AspNet.Mvc.Core.Test/Rendering/DefaultDisplayTemplatesTest.cs

```diff
@@ -29,17 +29,17 @@ public class DefaultDisplayTemplatesTest
                     { "<blink>text</blink>", false, "<blink>text</blink>" },
                     { "<blink>text</blink>", true, "&lt;blink&gt;text&lt;/blink&gt;" },
                     { "&'\"", false, "&'\"" },
-                    { "&'\"", true, "&amp;&#39;&quot;" },
```

Comment:
If you're unit-testing, don't rely on the default implementation of the encoders, as they're subject to change. Consider instead using a test encoder, such as the below.